go.mod

```diff
@@ -286,14 +233,14 @@ require (
 	go.opencensus.io v0.24.0 // indirect
 	go.uber.org/atomic v1.11.0 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
-	golang.org/x/exp v0.0.0-20240604190554-fc45aab8b7f8
-	golang.org/x/mod v0.18.0 // indirect
-	golang.org/x/net v0.26.0 // indirect
-	golang.org/x/sys v0.21.0 // indirect
-	golang.org/x/term v0.21.0 // indirect
-	golang.org/x/text v0.16.0 // indirect
+	golang.org/x/exp v0.0.0-20240531132922-fd00a4e0eefc
+	golang.org/x/mod v0.17.0 // indirect
```

Comment:
Why did these downgrade? I'm pretty sure there was a mistake during `go mod tidy` here.

Reply:
@Skarlso I do not know what happened here.

Should I add them manually? Normally `go mod tidy` picks that up...

Reply:
Not sure, try running a go mod tidy. What caused this go mod change in the first place?

Reply:
@Skarlso I changed our Pulumi library. We now have a ESC SDK.